hypha/apply/funds/views.py

```diff
@@ -1230,9 +1235,29 @@ class ReviewerLeaderboardDetail(SingleTableMixin, ListView):
     template_name = 'funds/reviewer_leaderboard_detail.html'
 
     def get_context_data(self, **kwargs):
-        User = get_user_model()
         obj = User.objects.get(pk=self.kwargs.get('pk'))
         return super().get_context_data(object=obj, **kwargs)
 
     def get_table_data(self):
         return super().get_table_data().filter(author__reviewer_id=self.kwargs.get('pk')).select_related('submission')
+
+
+@method_decorator(staff_required, name='dispatch')
+class StaffAssignments(SingleTableMixin, ListView):
+    model = User
+    table_class = StaffAssignmentsTable
+    table_pagination = False
+    template_name = 'funds/staff_assignments.html'
+
+    def get_queryset(self):
+        # Only list staff.
+        return self.model.objects.staff()
+
+    def get_table_data(self):
+        reviewer_roles = ReviewerRole.objects.all().order_by('order')
+        reviewer_role_id_1 = reviewer_roles.first().id
+        reviewer_role_id_2 = reviewer_roles.last().id
```

Comment:
@Parbhat I would like to dynamically create a column for each reviewer role. OTF happens to have two but the system allows you to create an arbitrary number of them. Any neat way to do this?

Reply:
`django-tables2` Table has an [extra_columns](https://django-tables2.readthedocs.io/en/latest/pages/api-reference.html#table) parameter to dynamically add columns. As we use `SimpleTableMixin`, these can be included in [get_table_kwargs](https://github.com/jieter/django-tables2/blob/6f5b5b6b228c3fc3a2491b83fe2c20f0e744dd01/django_tables2/views.py#L126) method.